assets/js/theme/product/image-gallery.js

```diff
@@ -17,10 +17,8 @@ export default class ImageGallery {
     setMainImage(imgObj) {
         this.currentImage = _.clone(imgObj);
 
-        this.destroyImageZoom();
         this.setActiveThumb();
         this.swapMainImage();
-        this.setImageZoom();
```

Comment:
i think we still need to call `setImageZoom`.  otherwise `this.easyzoom` will be undefined in `swapMainImage`, right?

Reply:
I don't believe we need to, as this zoom library works a bit differently than the old one. Now you just initialize it on an element (in this case the productView-image figure), and it manages swapping *both* the zoomed image and the normal-sized images. So the same easyzoom instance is actually sticking around as you swap images around.

Reply:
ah, i see - i completely missed that it's called from the `init` method.